tracer/src/Datadog.Trace/DatabaseMonitoring/DatabaseMonitoringPropagator.cs

```diff
@@ -73,5 +82,74 @@ internal static string PropagateSpanData(DbmPropagationLevel propagationStyle, s
 
             return string.Empty;
         }
+
+        /// <summary>
+        /// Uses a sql instruction to set a context for the current connection, bearing the span ID and trace ID.
+        /// This is meant to circumvent cache invalidation issues that occur when those values are injected in comment.
+        /// Currently only working for MSSQL (uses an instruction that is specific to it)
+        /// </summary>
+        /// <returns>True if the traceparent information was set</returns>
+        internal static bool PropagateDataViaContext(DbmPropagationLevel propagationLevel, IntegrationId integrationId, IDbConnection? connection, Span span)
+        {
+            if (propagationLevel != DbmPropagationLevel.Full || integrationId != IntegrationId.SqlClient || connection == null)
+            {
+                return false;
+            }
+
+            var stopwatch = System.Diagnostics.Stopwatch.StartNew();
+
+            byte version = 0; // version can have a maximum value of 15 in the current format
```

Comment:
This can be a constant.
```suggestion
            const byte version = 0; // version can have a maximum value of 15 in the current format
```